app/debug_overlay.conf

```diff
@@ -1 +1,6 @@
 CONFIG_DEBUG=y
+CONFIG_ASSERT=y
+CONFIG_STACK_SENTINEL=y
+CONFIG_SYS_HEAP_VALIDATE=y
```

Comment:
for HEAP_VALIDATE, sof does not call it directly, no sure zephyr will call it or not.
My feeling is that this does not like stack, low rate to have problems for heap.
I mean this maybe treated as option.

Stack overflow is must.

Reply:
@btian1 This has found issues in the past and it triggers an assert if corruption is hit.